spec/assertions_spec.cr

```diff
@@ -43,6 +43,12 @@ describe "Assertions" do
       it "should fail if the code block does not raise any exceptions" do
         assert_raise(Crotest::AssertionFailed) { assert_raise { } }
       end
+
+      it "returns the exception object" do
+        ex = assert_raise { raise "Boom!" }
+        assert_equal Exception, ex.class
+        assert_equal "Boom!", ex.to_s
+      end
```

Comment:
Probably, you want to test both cases, when you specify the Exception class and using the default.